### ISSUE TITLE
Adding City to Browse Client Files in CC display

### ISSUE THREAD

**Issue by drodruck** [josephr5000/confdb-public#238]
In one of the phone/conference (or the Friday) conversations we all had you (Joseph) mentioned that it would be helpful to add the city to the CC Browse Client Files display to help them determine they had the right conference.  It would also help to make sure we are looking at just the King County files, too.  Would it be possible to do that taking into consideration room on the screen?

**Comment by josephr5000** [comment]
Good suggestion.  This is in the live site and SHOULD be resurrected in the new one.  It can be squeezed in the address column, eg: "223 Yesler Way, Seattle".  Added to backlog.

**Comment by josephr5000** [comment]
FIXED

I compared having a separate column with appending it to the existing address column.  I decided to go with the ONE COLUMN approach as that is more space-compact, and city struck me as not being so important as to merit its very own column at the expense of line-wrapping other columns sooner.

Here are the two options.  Let me know if you would have traded this off differently.  To me neither one was a clear favorite.


![image](https://cloud.githubusercontent.com/assets/8585358/19118355/049dbf00-8ad1-11e6-9ebe-6c1aecb463f6.png)

![image](https://cloud.githubusercontent.com/assets/8585358/19118387/15eb3bf2-8ad1-11e6-8953-4842cccb1266.png)
